src/views/app/detail-panel/edit/parts/recipients-rows.tsx

```diff
@@ -22,6 +24,18 @@ export type RecipientsRowsProps = {
 	editorId: MailsEditorV2['id'];
 };
 
+function getExtraAccountsIds(): Array<string> {
```

Comment:
I think it makes sense to promote the function as an helper function and create a unit test for it

Reply:
i was undecided, thanks. moving to helpers folder